<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatchBasedInpaintingViewer</class>
 <widget class="QMainWindow" name="PatchBasedInpaintingViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>784</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Patch Based Inpainting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,6,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,9">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QRadioButton" name="radDisplayColorImage">
            <property name="text">
             <string>Color</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radDisplayMagnitudeImage">
            <property name="text">
             <string>Magnitude</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radDisplayChannel">
            <property name="text">
             <string>Channel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinChannelToDisplay"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QVTKWidget" name="qvtkWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Target</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="gfxTarget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="gfxSource">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Result</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="gfxResult">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLeft_Pane">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionFlipImageVertically"/>
    <addaction name="actionFlipImageHorizontally"/>
    <addaction name="actionSaveResult"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuLeft_Pane"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSavePointsLeft">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionSavePointsRight">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionOpenImageRight">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionLoadPointsLeft">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionLoadPointsRight">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenPointCloudLeft">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionOpenPointCloudRight">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionFlipImageVertically">
   <property name="text">
    <string>Flip Image Vertically</string>
   </property>
  </action>
  <action name="actionFlipRightImage">
   <property name="text">
    <string>Flip Right Image</string>
   </property>
  </action>
  <action name="actionSaveResult">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionOpenMask">
   <property name="text">
    <string>Open Mask</string>
   </property>
  </action>
  <action name="actionOpenMaskInverted">
   <property name="text">
    <string>Open Mask Inverted</string>
   </property>
  </action>
  <action name="actionFlipImageHorizontally">
   <property name="text">
    <string>Flip Image Horizontally</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
